app/renderer/components/tabs/content/favIcon.js

```diff
@@ -52,45 +52,40 @@ class Favicon extends React.Component {
       return null
     }
 
-    const iconStyles = StyleSheet.create({
-      icon__loading_color: {
-        filter: this.props.tabIconColor === 'white'
-          ? filter.makeWhite
-          : 'none'
-      },
-      icon__favicon: {
-        backgroundImage: `url(${this.props.favicon})`,
-        filter: this.props.tabIconColor === 'white'
-          ? filter.whiteShadow
-          : 'none'
-      },
-      icon__default_sizeAndColor: {
-        WebkitMaskSize: this.props.showIconAtReducedSize ? '10px' : '12px',
-        backgroundColor: this.props.tabIconColor === 'white'
-          ? color.white100
-          : color.mediumGray
-      }
-    })
+    const themeLight = this.props.tabIconColor === 'white'
+    const instanceStyles = { }
+    if (this.props.favicon) {
+      instanceStyles['--faviconsrc'] = `url(${this.props.favicon})`
+    }
 
     return <TabIcon
       data-test-favicon={this.props.favicon}
       data-test-id={this.testingIcon}
       className={css(
         styles.icon,
-        this.props.favicon && iconStyles.icon__favicon,
+        this.props.favicon && styles.icon_favicon,
+        this.props.favicon && themeLight && styles.icon_favicon__light,
```

Comment:
Should it not be `styles.icon_favicon_light`?

Reply:
@luixxiul yes, my mind getting confused with the different characters for bem that we're using! I've just updated all the names and think that they're conforming now so would be interested in what you think of the latest - thanks!

Reply:
maybe we'd create a bit more complicated example on https://github.com/brave/browser-laptop/blob/master/docs/style.md#defining-our-blocks-elements-and-modifiers ;-)

Reply:
The other names look good to me :-)